opentelemetry-instrumentation/README.rst

```diff
@@ -68,11 +68,13 @@ You can pass multiple values to configure multiple exporters e.g, ``zipkin,prome
 
 Well known trace exporter names:
 
-    - jaeger
+    - jaeger_proto
+    - jaeger_thrift
     - opencensus
     - otlp
-    - otlp_proto_grpc_span
```

Comment:
We have [proto](https://github.com/open-telemetry/opentelemetry-python/blob/main/exporter/opentelemetry-exporter-otlp-proto-grpc/setup.cfg#L58) exporter right?

Reply:
Good catch, that's correct, `otlp_proto_grpc_span` should stay.